python/sglang/srt/layers/moe/fused_moe_triton/fused_moe.py

```diff
@@ -49,6 +49,7 @@
 elif _is_cpu and _is_cpu_amx_available:
     pass
 elif _is_hip:
+    from sgl_kernel import gelu_and_mul, silu_and_mul
     from vllm import _custom_ops as vllm_ops  # gelu_and_mul, silu_and_mul
```

Comment:
Shall we remove `from vllm import _custom_ops as vllm_ops` for `moe_sum` or only add this when `_is_hip==True and _is_aiter==False`? Since MI250X does not have full aiter supports I think.